src/Components/CreateImageWizard/validators.ts

```diff
@@ -79,6 +85,41 @@ export const isSshKeyValid = (sshKey: string) => {
   return isPatternValid;
 };
 
+interface PasswordValidationResult {
```

Comment:
Didn't we decide to use `type` instead of `interface` just for the sake of consistency?